### PR TITLE
[EDA-844] [EDA-859]: Windows support for GTKWave

### PR DESCRIPTION
**Pull request by Nairi-RS**
### Motivate of the pull request
- [x] To address an existing issue. If so, please provide a link to the issue: RS issue
- [ ] Breaking new feature. If so, please describe details in the description part.

### Describe the technical details
Fixed issues related to Windows support.

### Which part of the code base require a change

- [ ] Library: <Specify the library name>
- [ ] Plug-in: <Specify the plugin name>
- [ ] Engine
- [ ] Documentation
- [ ] Regression tests
- [ ] Continous Integration (CI) scripts
- [x] CMake

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -280,12 +280,22 @@ add_custom_command(TARGET foedag POST_BUILD
...
-execute_process(COMMAND git config --global --add safe.directory ${CMAKE_CURRENT_SOURCE_DIR}
-  COMMAND git rev-parse --short HEAD
-  COMMAND tr -d '\n'
-  OUTPUT_VARIABLE GIT_HASH
-  ERROR_VARIABLE GIT_HASH_ERR
-  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+if(WIN32)
+    execute_process(COMMAND git config --global --add safe.directory ${CMAKE_CURRENT_SOURCE_DIR}
+      COMMAND git rev-parse --short HEAD
+      COMMAND dos2unix
```
Comment (ravic-rs) [inline CMakeLists.txt:286]: Why do we need this? Git should ideally handle the line endings by depending on the OS.

**Review by ravic-rs — COMMENTED** [review]
1 comment to address.

File: CMakeLists.txt (hunk 1 above)
Comment (aram-rs) [reply CMakeLists.txt:286]: Hi @ravic-rs , in our local setup without this we are getting empty git hash value in the FOEDAG_GIT_HASH define, which results to a compilation error. Also we were testing the build from Raptor level as well.

**Comment by aram-rs** [comment]
> @Nairi-RS what issues are we fixing in this PR? It needs to be filled into the description.

@ravic-rs it is a RS Jira, is it OK to provide the link here ?

File: CMakeLists.txt (hunk 1 above)
Comment (ravic-rs) [reply CMakeLists.txt:286]: This is interesting, will come back to this later.

**Pull request merged**